net/ssh/client.go

```diff
@@ -72,8 +72,9 @@ func ConnectToSSHServer(keys []ssh.Signer) (*ssh.Client, ssh.Signer, error) {
 
 func ConnectToSSHServerWithKey(key ssh.Signer) (*ssh.Client, error) {
 	sshConfig := &ssh.ClientConfig{
-		User: "git",
-		Auth: []ssh.AuthMethod{ssh.PublicKeys(key)},
+		User:            "git",
+		Auth:            []ssh.AuthMethod{ssh.PublicKeys(key)},
+		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
```

Comment:
From the documentaiton ( https://godoc.org/golang.org/x/crypto/ssh):

> InsecureIgnoreHostKey [...] should not be used for production code. 

Are you sure?...

Reply:
Adding this flag is the exact same level of security as before updating the lib.